services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -69,8 +84,42 @@ export const ReportPageWrapper = () => {
     setCurrentPageId(parentPage.childPageIds[newPageIndex]);
   };
 
+  const saveStatusText = "Last saved " + lastSavedTime;
+
   return (
     <FormProvider {...methods}>
+      <Flex sx={sx.subnavBar}>
```

Comment:
This whole chunk feels like it should become its own component, we're bloating the pageWrapper a bit.